src/services/services.ts

```diff
@@ -1079,6 +1079,7 @@ module ts {
 
     export interface CompletionInfo {
         isMemberCompletion: boolean;
+        isBuilder: boolean;

```

Comment:
Yeah, I still don't know what this is apart from a term that came up as a conceptual solution for the issue this addresses.